oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/management/ui/template/UITemplateInitializer.java

```diff
@@ -35,9 +35,9 @@
 import org.apache.skywalking.oap.server.library.util.StringUtil;
 
 /**
- * UITemplateInitializer load the template from the config file in json format. It depends on the UI implementation only.
- * Each config file should be only one dashboard setting json object.
- * The dashboard names should be different in the same Layer and entity.
+ * UITemplateInitializer load the template from the config file in json format. It depends on the UI implementation
+ * only. Each config file should be only one dashboard setting json object. The dashboard names should be different in
```

Comment:
You should import the latest code style, comments would not be re-formatted anymore.